appengine/flexible/AntiForgery/README.md

```diff
@@ -0,0 +1,54 @@
+# AntiForgery Sample for Google App Engine Flexible Environment
+
+This sample application is ready to deploy to Google App Engine Flexible Environment.
+
+## Prerequisites
+
+1.  **Follow the set-up instructions in [the documentation](https://cloud.google.com/dotnet/docs/setup).**
+
+2.  Install the [Google Cloud SDK](https://cloud.google.com/sdk/).  The Google Cloud SDK
+    is required to deploy .NET applications to App Engine.
+
+3.  Install the [.NET Core SDK, version 2.0](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.5-download.md)
+    or newer.
+
+
+## ![PowerShell](../.resources/powershell.png) Using PowerShell
+
+### Run Locally
+
+```psm1
+PS > dotnet restore
+PS > dotnet run
+```
+
+### Deploy to App Engine
+
+```psm1
+PS > dotnet restore
+PS > dotnet publish
+PS > gcloud beta app deploy .\bin\Debug\netcoreapp2.1\publish\app.yaml
+```
+
+
+## ![Visual Studio](../.resources/visual-studio.png) Using Visual Studio 2017
+
+Visual Studio is *optional*.  An old, unmaintained branch of samples that work
+with Visual Studio 2015 is
+[here](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/tree/vs2015).
```

Comment:
Possibly remove this paragraph as presumably this sample isn't in that branch, and using .NET Core with VS2015 is likely to be challenging?

Reply:
Done.